packages/core/apps/core-apps-server-internal/src/core_app.ts

```diff
@@ -249,6 +249,7 @@ export class CoreAppsService {
                 this.configService.setDynamicConfigOverrides(persistedOverrides.attributes);
                 latestOverrideVersion = persistedOverrides.version;
               }
+              this.logger.info('Succeeded in applying persisted dynamic config overrides');
```

Comment:
I'd move this log info inside the IF to only log when a change is actually applied. Otherwise, we'll log it every poll-interval.

Reply:
oh that makes perfect sense

Reply:
https://github.com/elastic/kibana/pull/197036/commits/3a1d5d57e6d022bbcb6b76bd07bc50c1a3b75c91